src/components/W2EmployerInfo.tsx

```diff
@@ -77,7 +77,7 @@ export default function W2EmployerInfo ({ navButtons, onAdvance }) {
           register={register}
           required={true}
           name="employerCity"
-          defaultValue={prevFormData.employerCity}
+          defaultValue={prevFormData?.employerCity ?? ''}
```

Comment:
Does defaultValue need to be a string? defaultValue could be set as an optionally defined string in the labeledInputProps interface like so 
```
interface LabeledInputProps {
  ...
  defaultValue?: string
}
```

Reply:
Ah very nice.